src/include/general.h

```diff
@@ -76,7 +76,12 @@
 #define PRINT_NOOP(...) \
 	do {                \
 	} while (false)
-#if defined(ENABLE_DEBUG)
+
+#ifndef ENABLE_DEBUG
+#define ENABLE_DEBUG 0
```

Comment:
It would be nice if we could pass this via the build system when it shouldn't be true.

Reply:
You can, see https://github.com/blackmagic-debug/blackmagic/pull/1679/files#diff-3e2513390df543315686d7c85bd901ca9256268970032298815d2f893a9f0685L24-R31

Just set `ENABLE_DEBUG != 1`

This is just a safe guard.

Reply:
Think it's entirely safe to drop this then in favour of the build system - but we'll keep it in for this PR so that, eg, Farpatch can catch up to the build system changes here.